pkg/apis/clusteroperator/types.go

```diff
@@ -406,16 +406,8 @@ type ClusterProviderStatus struct {
 
 // ClusterDeploymentStatus contains the status for a cluster
 type ClusterDeploymentStatus struct {
-	// Ready is true if the master of the cluster is ready to be used and can be accessed using
```

Comment:
Ignore the previous comment to remove this :) ... I'm looking at this commit by commit